theme.json

```diff
@@ -34,49 +191,179 @@
 			"gradients": [],
 			"palette": [
 				{
-					"color": "#425a5e",
+					"color": "#EBE9FB",
+					"slug": "primary-50",
+					"name": "Primary 50"
+				},
+				{
+					"color": "#D7D3F6",
 					"slug": "primary-100",
 					"name": "Primary 100"
 				},
 				{
-					"color": "#2d3d40",
+					"color": "#AEA6EE",
 					"slug": "primary-200",
 					"name": "Primary 200"
 				},
 				{
-					"color": "#3f4040",
+					"color": "#867AE5",
 					"slug": "primary-300",
 					"name": "Primary 300"
 				},
 				{
-					"color": "#fff",
+					"color": "#5E4DDC",
+					"slug": "primary-400",
+					"name": "Primary 400"
+				},
+				{
+					"color": "#3B28CC",
+					"slug": "primary-500",
+					"name": "Primary 500"
+				},
+				{
+					"color": "#2F20A3",
+					"slug": "primary-600",
+					"name": "Primary 600"
+				},
+				{
+					"color": "#23187A",
+					"slug": "primary-700",
+					"name": "Primary 700"
+				},
+				{
+					"color": "#E9F0FF",
+					"slug": "secondary-50",
+					"name": "Secondary 50"
+				},
+				{
+					"color": "#D4E1FF",
 					"slug": "secondary-100",
 					"name": "Secondary 100"
 				},
 				{
-					"color": "#e3e4eb",
+					"color": "#A8C2FF",
 					"slug": "secondary-200",
 					"name": "Secondary 200"
 				},
 				{
-					"color": "#d3d3d3",
+					"color": "#7DA4FF",
 					"slug": "secondary-300",
 					"name": "Secondary 300"
 				},
 				{
-					"color": "#f48d60",
+					"color": "#5185FF",
+					"slug": "secondary-400",
+					"name": "Secondary 400"
+				},
+				{
+					"color": "#2667FF",
+					"slug": "secondary-500",
+					"name": "Secondary 500"
+				},
+				{
+					"color": "#0046EA",
+					"slug": "secondary-600",
+					"name": "Secondary 600"
+				},
+				{
+					"color": "#0035B0",
+					"slug": "secondary-700",
+					"name": "Secondary 700"
+				},
+				{
+					"color": "#E9F4FD",
+					"slug": "tertiary-50",
+					"name": "Tertiary 50"
+				},
+				{
+					"color": "#D3EAFA",
 					"slug": "tertiary-100",
 					"name": "Tertiary 100"
 				},
 				{
-					"color": "#f47c48",
+					"color": "#A8D5F5",
 					"slug": "tertiary-200",
 					"name": "Tertiary 200"
 				},
 				{
-					"color": "#f3713c",
+					"color": "#7CBFF1",
```

Comment:
This color is being used for links and with a contrast ratio of 1.98 it's not accessible.

<img width="174" alt="screen-capture 2023-08-18 at 11 56 27 AM" src="https://github.com/WebDevStudios/wd_f/assets/12365909/57d170af-494c-40f2-8820-b9c5514ef9cd">

Also, I noticed links have this color in the editor but not on the frontend.

<img width="2018" alt="screen-capture 2023-08-18 at 11 59 37 AM" src="https://github.com/WebDevStudios/wd_f/assets/12365909/fef7076c-4550-44f8-871f-fc0c590b121a">

Reply:
This was a mistake :P The additional colors should not have been added in, reverting to just the default from before.